### PR TITLE
Add Raspberry Pi 3 support

### PR DESCRIPTION
**Pull request by erikboto**
I'll look through this an extra round tomorrow, but am grateful for any early input

### CODE REVIEW

**Comment by erikboto** [comment]
Oh right, was working on personal laptop on train :) Will fix!
mån 15 maj 2017 kl. 16:51 skrev Gordan Markuš <notifications@github.com>:

File: getting-started.md [hunk 1]
```diff
@@ -26,10 +27,19 @@ $HOME/pelux-baseline/sources/poky/scripts/contrib/mkefidisk.sh <host-device> $HO
...
+### Raspberry Pi 3
+Write the generated image file to an SD-card using e.g.:
+dd if=$HOME/pelux-baseline/build/tmp/deploy/images/raspberrypi3/core-image-pelux-raspberrypi3.rpi-sdimg of=<host-device> bs=4M
+
+Where <host-device> is the SD-card device on the host, e.g. /dev/mmcblk0 or /dev/sdc.
+
 Booting on target hardware
 --------------------------
 
 ### NUC
```
Comment (DunderRoffe) [inline getting-started.md]: Should probably say NUC / Minnowboard

File: getting-started.md [hunk 2]
```diff
@@ -12,11 +12,12 @@ These instructions assume there is an image built according to the
 
 * The top level of the build is located in `$HOME/pelux-baseline`
 * In that top level directory there is a directory named `build`
-* The `pelux-intel.xml` manifest was used
+* The `pelux-intel.xml` or `pelux-rpi.xml` manifest was used
 * The `core-image-pelux` image was built
 
 Writing the image
 -----------------
+### NUC / minnowboard
```
Comment (DunderRoffe) [inline getting-started.md]: minnowboard -> Minnowboard

Comment (erikboto) [inline getting-started.md]: Will fix!

File: getting-started.md (hunk 1 above)
Comment (erikboto) [inline getting-started.md]: Fixed, but this wasn't really changed by my PR :)

**Comment by erikboto** [comment]
See https://github.com/Pelagicore/pelux-manifests/pull/40

**Pull request closed without merging**